src/Flashcards.js

```diff
@@ -26,21 +27,28 @@ class Flashcards extends Component {
     const currentFlashcard = flashcards[selectedFlashcard];
 
     return (
-      <div className='Flashcards-content'>
-        <div className='Flashcards-button'>
-          <Button icon='keyboard_arrow_left' floating
-            disabled={selectedFlashcard === 0}
-            onClick={onClickPreviousFlashcard}
+      <div className='Flashcards'>
```

Comment:
We already have a `div` with a class name `Flashcards`. Look below in the `render` method.